### PR TITLE
dont exchange handle, just change the algorithm of track

### PR DESCRIPTION
**Pull request by Robin-front**
trace: #298 

about the wrong handle focus, I think we could try to keep the same handle instead of exchange handles. just change the algorithm of track for display correctly.

**but it will cause the handle reverse, no lowerhandle and upperhandle concept. it's potential risk.**

if you agree that, #300 should also make a change in `props.allowCross` logic.

### CODE REVIEW

File: src/Range.jsx [hunk 1]
```diff
@@ -123,13 +123,10 @@ class Range extends React.Component {
 
     const nextBounds = [...state.bounds];
     nextBounds[state.handle] = value;
-    let nextHandle = state.handle;
+    const nextHandle = state.handle;
     if (props.pushable !== false) {
       const originalValue = state.bounds[nextHandle];
       this.pushSurroundingHandles(nextBounds, nextHandle, originalValue);
-    } else if (props.allowCross) {
```
Comment (benjycui) [inline src/Range.jsx]: You are removing a feature?!

Comment (Robin-front) [inline src/Range.jsx]: I think no.
the codes about `allowCross` feature is here [https://github.com/react-component/slider/blob/master/src/Range.jsx#L270](https://github.com/react-component/slider/blob/master/src/Range.jsx#L270)
where I delete is update handle controller, isn't it ..

**Comment by paranoidjk** [comment]
@RaoHai have you test with our example? Obviously there have some problem, the range value will be something like `[60, 50]` if lowerhandle across upperhandle.

Please read our dev doc https://github.com/react-component/slider#development and test more carefully. 😄

**Comment by Robin-front** [comment]
@paranoidjk yes, I have to admit that there is a problem. That's why I said no lowerhandle and upperhandle concept.  If just the range output value is reverse, I think you can sort before output.

But, obviously, it's no the best resolvent. 
and maybe it will cause other problem.
e.g. order of focus is wrong because the dom order didn't change.
cross

**Pull request closed without merging**